starfyre/compiler.py

```diff
@@ -2,13 +2,24 @@
 from pathlib import Path
 
 
-def get_fyre_files(project_dir):
+def get_fyre_files(project_dir):   
     fyre_files = []
     for file in os.listdir(project_dir):
         if file.endswith(".fyre"):
             fyre_files.append(file)
     return fyre_files
 
+def import_css(import_statement):
+    """Read a css file and save it's content to a list"""
+    css_content = []    
+    css_file_name = import_statement.split("'")[1]  
```

Comment:
This statement assumes that we only have import statements like `import 'test.css'`, but we would also want to support `import "test.css"`.


You can try a regular expression based approach here.

Something like this

```
import re

pattern = re.compile(r'import\s+["\'](.*?)\.css["\']')
match = pattern.search(statement1)
filename = match.group(1)
```

Reply:
Or maybe you can use the Path(https://docs.python.org/3/library/pathlib.html) function to resolve the path here. 

This would be the preferred approach but we can resort to regex if this doesn't work.

Reply:
Thank you for the suggestion. I implemented a simpler way to support both statements for now. Please take a look and let me know what you think. :)